apps/server/src/modules/common-cartridge/export/elements/v1.1.0/common-cartridge-element-factory.spec.ts

```diff
@@ -4,7 +4,7 @@ import {
 	createCommonCartridgeOrganizationElementPropsV110,
 	createCommonCartridgeOrganizationsWrapperElementPropsV110,
 	createCommonCartridgeResourcesWrapperElementPropsV110,
-} from '@shared/testing/factory/common-cartridge-element-props.factory';
+} from '@src/modules/common-cartridge/testing/common-cartridge-element-props.factory';
```

Comment:
Please use relativer path inside the same module ../../../testing/common-cartridge-element-props.factory
For all other places too.

Reply:
Done anywhere.